frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -377,6 +377,14 @@ interactivetour-InteractiveTour--history-content =
     each entry indicate its review status (Approved, Rejected or
     Unreviewed).
 
+interactivetour-InteractiveTour--terms-title = Terms
+interactivetour-InteractiveTour--terms-content =
+    Test
+
+interactivetour-InteractiveTour--comments-title = Comments
+interactivetour-InteractiveTour--comments-content =
+    Test
```

Comment:
_In the Comments tab you can discuss how to translate a string with your fellow team members. It's also the place where you can request more context about or report an issue in the source string._